src/backend/aspen/app/views/api_utils.py

```diff
@@ -189,6 +189,9 @@ def authz_sample_filters(query: Query, sample_ids: Set[str], user: User) -> Quer
             ),
         )
     )
+    query = query.filter(
+        and_(or_(~Sample.private, Sample.submitting_group_id == user.group_id))
+    )
```

Comment:
so this is different level of privacy aside from just being able to see private_identifiers? I guess i'm a little confused why we need this. i thought the `private` field was an indicator for if a sample didn't have an associated gisaid id?

Reply:
According to the privacy policy, it's a flag to prevent sharing samples with outside groups that normally have access to your group's samples (e.g: cdph)